engine/src/main/java/org/terasology/engine/audio/openAL/OpenALManager.java

```diff
@@ -260,7 +261,7 @@ private void notifyEndListeners(boolean interrupted) {
                 try {
                     entry.getValue().onAudioEnd(interrupted);
                 } catch (Exception e) {
-                    logger.error("onAudioEnd() notification failed for {}", entry.getValue(), e);
+                    logger.atError().addArgument(() -> entry.getValue()).addArgument(e).log("onAudioEnd() notification failed for {}");
```

Comment:
The associated method will only be evaluated if the catch clause is evaluated, so I think we do not need the fluent API here. Please ignore PMD warnings for this line instead:
```suggestion
                    logger.error("onAudioEnd() notification failed for {}", entry.getValue(), e); //NOPMD
```

Reply:
fixed

Reply:
this doesn't seem to be fixed :thinking: